opmd_viewer/openpmd_timeseries/data_reader/utilities.py

```diff
@@ -124,3 +124,34 @@ def get_shape(dset):
         shape = dset.shape
 
     return(shape)
+
+
+def join(base_path, *paths):
```

Comment:
In the end I implemented a simple join method which I think should be enough for this case.

Reply:
I think that's a great way to do it!
One very minor comment: could you name this function `filepath_join` instead of `join`, so that the name tells a bit more about what the function does.